src/chunkserver/network_worker_thread.cc

```diff
@@ -1494,29 +1550,28 @@ void worker_write(csserventry *eptr) {
 			stats_bytesout += (bytesInBufferBefore - bytesInBufferAfter);
 			if (ret == OutputBuffer::WRITE_ERROR) {
 				safs_silent_errlog(LOG_NOTICE, "(write) write error");
-				eptr->state = CLOSE;
+				eptr->state = ChunkserverEntry::State::Close;
 				return;
 			} else if (ret == OutputBuffer::WRITE_AGAIN) {
 				return;
 			}
 		} else {
-			i = write(eptr->sock, pack->startptr, pack->bytesleft);
+			i = write(eptr->sock, pack->startPtr, pack->bytesLeft);
```

Comment:
Same here.

Reply:
Applied here: https://github.com/leil-io/saunafs/commit/8f8dcf733ccd489fa6b9b4426b1ec5629500c839
Please notice this is out of the scope of this change.